edx-platform/pearson-bsg-theme/lms/templates/index.html

```diff
@@ -6,39 +6,32 @@
 from django.urls import reverse
 
 from openedx.core.djangolib.markup import HTML, Text
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+%>
+<%
+home_page_title = configuration_helpers.get_value('home_page_title', 'Welcome')
+home_page_subtitle = configuration_helpers.get_value('home_page_subtitle', 'We think you might be interested in these programs')
+banner_image_url = configuration_helpers.get_value('banner_image_url', '/static/pearson-pols-theme/images/banner-pols.jpg')
```

Comment:
These variables are only used once, so it is not necessary to define them, we can use them directly in the parts that they are being used. @amalbas

Reply:
@Squirrel18 done